index.js

```diff
@@ -0,0 +1,120 @@
+const inquirer = require("inquirer");
+const figlet = require('figlet');
+const mysql = require("mysql2");
+const cTable = require("console.table");
+
+const connect = require("./db/db");
+var db;
+
+const ALL_CLASSES = {
+  Employee: require("./lib/Employee"),
```

Comment:
interesting approach using classes! I've never seen this assignment completed this way, great thiinking :)